### PR TITLE
when `skip` is specified include it in the set of projects to clone but set all commands to be empty

### LINKED ISSUE #83: `skip` option should add projects while constructing tree
Currently `skip` option does not add the project to the tree. However, we want the project to still be added and cloned but just not executed

### PR DESCRIPTION
**Pull request by shubhbapna**
Fixes #83

### CODE REVIEW

File: src/util/construct-tree.ts [hunk 1]
```diff
@@ -57,26 +57,25 @@ function constructNode(
...
-        after: buildCommand?.after,
-        commands: {
-          upstream: buildCommand?.upstream ?? [],
-          downstream: buildCommand?.downstream ?? [],
-          current: buildCommand?.current ?? [],
-        },
-        ...(clone ? { clone } : {}),
-      };
+  const skip = buildConfig?.skip;
+  const buildCommand = !skip ? getBuildCommand(dependency.project, defaultBuild, build) : undefined;
```
Comment (Ginxo) [inline src/util/construct-tree.ts:63]:
```suggestion
  const buildCommand = !buildConfig?.skip ? getBuildCommand(dependency.project, defaultBuild, build) : undefined;
```
why not?

**Review by Ginxo — APPROVED** [review]
LGTM, just technical question

File: src/util/construct-tree.ts (hunk 1 above)
Comment (shubhbapna) [reply src/util/construct-tree.ts:63]: woops you are right i was copy pasting stuff and missed this

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**